src/main/java/seedu/address/logic/commands/CreateTaskCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK;
+
+import java.util.HashSet;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * Adds a Task to the address book.
+ */
+public class CreateTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "create-task";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a task in the address book. \n"
+            + "Parameters: "
+            + PREFIX_TASK + "TASK_TYPE TASK_DESCRIPTION [ADDITIONAL_FIELDS]\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TASK + "todo Setup venue decorations\n"
+            + COMMAND_WORD + " "
+            + PREFIX_TASK + "deadline Submit proposal /by 2024-10-31";
```

Comment:
In my implementation of Wedding date, I'm using a `d/` prefix. Maybe we can share that instead

Reply:
I will be changing the implementation of tasks such that the user won't have to specify task types, but if they specify one date it will be checked as a Deadline, and two dates as an Event, so yes I will change it to use the d/ prefix in the next milestone, thanks